### PR TITLE
Fix func upgrade_cluster_version & check_cluster_version_operator_health

### PR DESCRIPTION
**Pull request by kasturinarra**
Fix function upgrade_cluster_version & check_cluster_version_operator_health so that ocp upgrade could be performed.

### CODE REVIEW

**Comment by kasturinarra** [comment]
@btjd @Dannyb48 @adtandale @jbpratt could you please help review ? Thanks !!

File: piqe_ocp_lib/api/resources/ocp_cluster_versions.py [hunk 1]
```diff
@@ -170,12 +170,19 @@ def wait_until_upgraded(timeout: int):
...
             ends_at = datetime.now() + timedelta(minutes=timeout)
-            health_checker = OcpHealthChecker()
+            health_checker = OcpHealthChecker(self.kube_config_file)
+
+            cluster_version = self.get_cluster_version()
+            for condition in cluster_version["status"]["conditions"]:
+                if condition["type"] == "Progressing" and condition["status"] == "True":
+                    break
+                else:
+                    continue
```
Comment (jbpratt) [inline piqe_ocp_lib/api/resources/ocp_cluster_versions.py:180]: nit-picky: you can remove the `else` here

**Review by jbpratt — APPROVED** [review]
LGTM!

File: piqe_ocp_lib/api/resources/ocp_cluster_versions.py (hunk 1 above)
Comment (kasturinarra) [reply piqe_ocp_lib/api/resources/ocp_cluster_versions.py]: Done, thanks !!

**Review by Dannyb48 — APPROVED** [review]
LGTM

**Pull request merged**